p2p/src/network/noise/p2p_network_noise_state.rs

```diff
@@ -127,7 +127,8 @@ impl NoiseState {
         let hkdf = Hkdf::<Sha256, Hmac<Sha256>>::new(Some(&self.chaining_key.0), &secret);
         secret.zeroize();
         let mut okm = [0; 64];
-        hkdf.expand(&[], &mut okm).unwrap();
+        hkdf.expand(&[], &mut okm)
+            .expect("the length is constant and small");
```

Comment:
Is the `expect` here fine because sizes are hardcoded and can't break unless the code is changed? Could you add a brief comment explaining it?

Reply:
Yes, this method returns an `InvalidLength` error and this is invalid if the size is greater than some constant (255 times the block size). In our case the size is always 64. Even if the constant changes in the future, this will cause any test to fail. So we will not miss this.